pkg/migration/migration.go

```diff
@@ -302,7 +302,8 @@ storage:
 		return fail("unable to make configuration for the storage: %s", err)
 	}
 
-	tls, tlsSecret, err := migrateTLS(config, podFileGetter)
+	//tls, tlsSecret, err := migrateTLS(config, podFileGetter)
```

Comment:
Did you mean to just comment this out and not delete it? (same with a few lines below)

Reply:
@bparees Since you deleted the rest of the code and did not comment out, this comment does not make sense.